### PR TITLE
Add Japanese Translations to stringtable.xml

### PR DESCRIPTION
**Pull request by Apricot-ale**
Add Japanese Translations!
Super saiyan simulator 😂

### CODE REVIEW

File: addons/main/stringtable.xml [hunk 1]
```diff
@@ -3,171 +3,227 @@
...
 			<English>Maximum height that a player can drop from</English>
+			<Japanese>プレイヤーが飛び降りることのできるオブジェクトの高さの最大値です。</Japanese>
 		</Key>
 		<Key ID="STR_EMR_Main_SettingDisplayName_jumpVelocity">
 			<English>Jump velocity</English>
+			<Japanese>ジャンプの速度</Japanese>
 		</Key>
 		<Key ID="STR_EMR_Main_SettingDescription_jumpVelocity">
 			<English>Super saiyan simulator</English>
+			<Japanese>スーパーサイヤ人シミュレーター! \n(訳注:これはジャンプ時にどれだけ跳ねるかを設定します)</Japanese>
```
Comment (Apricot-ale) [inline addons/main/stringtable.xml:78]: As a translation, I wrote what kind of functions are included.

**Comment by SceptreOfficial** [comment]
Thanks! Feel free to add yourself to the 'authors.txt' as well. I'll leave this open for a day or so, in case you have any changes you want to make before I merge.

**Comment by SceptreOfficial** [comment]
I see a few updates. Let me know if you're done with any changes and I'll merge.

**Comment by SceptreOfficial** [comment]
Haven't seen any activity so I'm merging

**Comment by Apricot-ale** [comment]
> Haven't seen any activity so I'm merging

Sorry. I didn't report it.
It's already merged, but that's okay!

If new config/strings are added, I will add new translations.
Arigatogozaimasu. :)

**Comment by SceptreOfficial** [comment]
Sounds good, thank you!

**Pull request merged**